lib/stages.ts

```diff
@@ -137,79 +138,85 @@ export class LisaServeApplicationStage extends Stage {
 
     serveStack.addDependency(iamStack);
 
-    if (config.deployChat || config.deployUi || config.deployRag) {
-      const apiBaseStack = new LisaApiBaseStack(this, 'LisaApiBase', {
-        ...baseStackProps,
-        stackName: createCdkId([config.deploymentName, config.appName, 'API']),
-        description: `LISA-API: ${config.deploymentName}-${config.deploymentStage}`,
-        vpc: networkingStack.vpc.vpc,
-      });
-      apiBaseStack.addDependency(coreStack);
-      stacks.push(apiBaseStack);
```

Comment:
I think you need to rebase your changes with the current HEAD of develop. We *want* those changes there. This would undo some of the changes that were needed for our partner teams.

Reply:
Yeah I had a bad merge that I think some of this got messed up. I reverted this file to what it is in develop and just added back in my specific changes.